dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
Shouldn't this stuff already be in an experimental branch instead of master, anyway? The current expectation is that code that is submitted to master (like my current PR sitting in the queue) will make it into the next dnf-plugins-core release, and it will work with the current stack.

I know that I don't really have much say here, but I'm generally not comfortable with the idea of merging this into master while all the other stuff works fine with hawkey 0.6.x. I would suggest not merging this at all until libhif 0.7.0 is in a state where it can be released. It's a much easier pill to swallow, then.

And in all honesty, I would prefer to use libhif 0.7.0 in Mageia 6, but because I have no idea when that will be ready, I'm relying on the expectation that hawkey 0.6.3 will work for the time being.